executable/dex.ksy

```diff
@@ -631,7 +631,7 @@ types:
   code_item:
     doc-ref: 'https://source.android.com/devices/tech/dalvik/dex-format#code-item'
     seq:
-      - id: register_count
+      - id: registers_count
```

Comment:
It seems like the former variant was grammatically more correct. 

https://english.stackexchange.com/questions/11350/use-of-the-plural-with-list-and-when-item-count-is-0